rfc.md

```diff
@@ -0,0 +1,106 @@
+# PHP RFC: Nullsafe operator
+
+* Date: 2020-05-25
+* Author: Ilija Tovilo, tovilo.ilija@gmail.com
+* Status: Under discussion
+* Target Version: PHP 8.0
+* Implementation: <https://github.com/php/php-src/pull/5619>
+* Supersedes: <https://wiki.php.net/rfc/nullsafe_calls>
+
+## Introduction
+
+This RFC proposes a new operator nullsafe operator `?->` with full short-circuiting.
+
+## Proposal
+
+It is fairly common to only want to call a method or fetch a property on the result of an expression if it is not `null`. This is usually done with if statements / ternary expressions and temporary variables.
+
+```php
+$bar = $foo !== null ? $foo->bar() : null;
+$baz = $bar !== null ? $bar->baz : null;
+
+// or
+
+if ($foo === null) {
+    return;
+}
+$bar = $foo->bar();
+
+if ($bar === null) {
+    return;
+}
+$baz = $bar->baz;
+```
+
+With the nullsafe operator `?->` this code could instead be expressed
+like this:
+
+```php
+$baz = $foo?->bar()?->baz;
+```
+
+When the left hand side of the operator evaluates to `null` the execution of the entire chain will stop and evalute to `null`. When it is not `null` it will behave exactly like the normal `->` operator.
+
+### Short circuiting
+
+This RFC proposes full short circuiting. This means when the evaluation of one element in the chain fails the execution of the entire chain is aborted and the entire chain evaluates to `null`. The following elements are considert part of the chain.
```

Comment:
You have a typo: `considert` => `considered`

Reply:
Fixed, thanks!